packages/blade/src/components/Input/_decisions/decisions.md

```diff
@@ -0,0 +1,212 @@
+
+### Return Key types for web and Native
+
+| Key Type            | Web                       | Native                                   |
+| ------------------- | ------------------------- | ---------------------------------------- |
+| <kbd>↵</kbd>        | `enterkeyhint="enter"`    | NA                                       |
+| <kbd>go</kbd>       | `enterkeyhint="go"`       | `returnKeyType="go"`                     |
+| <kbd>Done</kbd>     | `enterkeyhint="done"`     | `returnKeyType="done"`                   |
+| <kbd>Next</kbd>     | `enterkeyhint="next"`     | `returnKeyType="next"`                   |
+| <kbd>Previous</kbd> | `enterkeyhint="previous"` | `returnKeyType="previous"`(android only) |
+| <kbd>Search</kbd>   | `enterkeyhint="search"`   | `returnKeyType="search"`                 |
+| <kbd>Send</kbd>     | `enterkeyhint="send"`     | `returnKeyType="send"`                   |
+
+## PasswordInput
+
+**Sample Usage**
+
+```tsx
+<PasswordInput
+  label="Enter Password"
+  labelPosition="top"
+  placeholder="Enter your password"
+  errorText="Password cannot be blank"
+  validationState={!inputValue.length ? 'error' : undefined}
+/>
+```
+
+| Prop                                                                                               | Type                              | Required                                                                            | Default                                                   | Description                                                                                                                                                                                                                                                                           |
+| -------------------------------------------------------------------------------------------------- | --------------------------------- | ----------------------------------------------------------------------------------- | --------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| label                                                                                              | `string`                          | (confirm if we have a use case for input field without labels(search field maybe?)) |                                                           | Determines the label of an input field. Also used as `aria-label`                                                                                                                                                                                                                     |
```

Comment:
we can make this required in other inputs other than `TextInput` right? For example, I don't see usecase of `PasswordInput` not requiring label.

Reply:
yes it was pending on design